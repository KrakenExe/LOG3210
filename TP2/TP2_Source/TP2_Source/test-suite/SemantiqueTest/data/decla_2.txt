Declaration
num a;
bool b;
num a;
EndDeclaration
