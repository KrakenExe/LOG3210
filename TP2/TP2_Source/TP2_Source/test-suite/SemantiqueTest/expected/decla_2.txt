Invalid declaration... variable a already exists